site/docs/extensions/filter-control.md

```diff
@@ -202,7 +192,43 @@ Dependence if you use the datepicker option: [bootstrap-datepicker](https://gith
 
 - **Detail:**
 
-   If the datepicker option is set use this option to configure the datepicker with the native options. Use this way: `data-filter-datepicker-options='{"autoclose":true, "clearBtn": true, "todayHighlight": true}'`.
+   If the datepicker option is set use this option to configure the datepicker with the native options. Use this way: `data-filter-datepicker-options='{"property1":value1, "property2": value2, "property3": value3}'`.
+
+- **Default:** `undefined`
+
+### filterControlMultipleSelect
```

Comment:
I dont understand why we need two functions to enable the multiple select.
Also the second option `filterControlMultipleSelectMultiple` description is the same as this one. Both says `use this option to configure the select as multiple select.`.

I'm also not able to use a multiple select, because i dont know that options i added both with `true` but there is no (multi)select in the header. (Example?)

Reply:
I searched in the code and could figure out why there are two options, but then we should have a better documentation and option names. It's quite confusing right now.

There is also a issue with the multiple select, you can quickly select a few values.
After that the table filters and the values in the multi select are no more select!

Example: https://live.bootstrap-table.com/code/UtechtDustin/7453
To reproduce it just select one (or more) values of the multi select, after the filtering is done the values are not selected!

Reply:
Going to review this

Reply:
It is fixed now. Remove the extra option.